Traceback (most recent call last):
  File "/hpc/home/csr33/ast_object_detection/images_for_prediction/naip_in_tracts.py", line 82, in <module>
    main(args)
  File "/hpc/home/csr33/ast_object_detection/images_for_prediction/naip_in_tracts.py", line 72, in main
    extracted_naip = extracted_naip + extract_naip_tiles_by_tract(polygon, date_range)
                                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/hpc/home/csr33/ast_object_detection/images_for_prediction/naip_in_tracts.py", line 34, in extract_naip_tiles_by_tract
    search = catalog.search(collections=["naip"], intersects=area_of_interest,
             ^^^^^^^
NameError: name 'catalog' is not defined
